base/install_area_detector.sh

```diff
@@ -66,13 +66,17 @@ WITH_OPENCV=NO
 WITH_SZIP=YES
 SZIP_EXTERNAL=NO
 
-XML2_EXTERNAL=NO
+XML2_EXTERNAL=YES
+XML2_LIB=/usr/lib/x86_64-linux-gnu
```

Comment:
I don't love that this hardcodes x86_64... Especially in light of us having talked to Marco about using this for mobipix.

I think you should be able to get something using `pkg-config` to print `libdir`

Reply:
At first, I didn't think libxml2 had `pkg-config` support. It turns out I mistook its name.

I've used `pkg-config` to determine the `XML2_INCLUDE` path as well. Same approach is used for GLib and Aravis.